controllers/workspace/metrics/update.go

```diff
@@ -34,7 +44,27 @@ func incrementMetricForWorkspace(metric *prometheus.CounterVec, wksp *dw.DevWork
 	sourceLabel := wksp.Labels[workspaceSourceLabel]
 	ctr, err := metric.GetMetricWith(map[string]string{metricSourceLabel: sourceLabel})
 	if err != nil {
-		log.Info("Failed to increment metric: %s", err)
+		log.Error(err, "Failed to increment metric")
 	}
 	ctr.Inc()
 }
+
+func incrementStartTimeBucketForWorkspace(wksp *dw.DevWorkspace, log logr.Logger) {
+	sourceLabel := wksp.Labels[workspaceSourceLabel]
+	hist, err := workspaceStartupTimesHist.GetMetricWith(map[string]string{metricSourceLabel: sourceLabel})
```

Comment:
So, we would need to create PR against OpenShift Console, and probably backport it to OS 4.8, so also update existing CR is Console if label is not detected.

Reply:
Done (finally): https://github.com/openshift/console/pull/9752